cms/grading/tasktypes/Batch.py

```diff
@@ -262,6 +265,7 @@ def evaluate(self, job, file_cacher):
             commands,
             job.time_limit,
             job.memory_limit,
+            max_processes,
```

Comment:
This is a little too ugly/unintuitive for mine.

The current default argument for max_processes is 1 -- it should be LANGUAGE_TO_MAX_PROCESSORS['default']. From there, just use
`num_processes = LANGUAGE_TO_MAX_PROCESSORS[language] if language in LANGUAGE_TO_MAX_PROCESSORS else LANGUAGE_TO_MAX_PROCESSORS['default']`, or similar; keeping `default_processes` and `max_processes` around is somewhat ugly and nonintuitive to follow, since you'll only use *one* of them.